.github/workflows/lint.yml

```diff
@@ -0,0 +1,81 @@
+name: Lint
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  run-eslint:
+    name: Run eslint
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+
+      - name: Install Node.js dependencies
+        run: yarn
+        env:
+          SKIP_PREPARE: true
+
+      - name: Run eslint
+        uses: reviewdog/action-eslint@v1
```

Comment:
Note that you don't need a custom action for this. Github will pick up the results of a simple `yarn lint`.

See https://github.com/jellyfin/jellyfin-vue/blob/master/.github/workflows/main.yml#L14-L49

Reply:
I had used the custom action because if we grant it additional permissions it will add code suggestions in comments, but this is fine for now since we don't have that set up yet.